packages/ui-library/src/components/select/index.ts

```diff
@@ -222,10 +208,10 @@ export class BlrSelect extends LitElementCustom {
                 })}
               </select>
             </div>
-            ${this.renderIcon(iconClasses)}
+            ${this.renderIcon()}
           </div>
           ${(this.hasHint && this.hintMessage) || (this.hasError && this.errorMessage)
-            ? BlrFormCaptionGroupRenderFunction({ sizeVariant: this.sizeVariant }, getCaptionContent())
+            ? BlrFormCaptionGroupRenderFunction({ sizeVariant: this.sizeVariant }, captionContent)
```

Comment:
In one of my recent previous PR I changed this to be a getCaptionContent function rather than just a const captionContent otherwise the captionContent const is always populated regardless of it gets used or not. Is there any reason why you changed it back?

Reply:
Sorry for the oversight. Will change it to it to a non-inline function. I changed it because creating an inline function on every render costs time too. I converted it out of reflex without thinking about if it's actually more performant in this case 😅